code/0_bootstrap.py

```diff
@@ -83,6 +83,10 @@
         root = tree.getroot()
         for prop in root.findall("property"):
             if prop.find("name").text == "hive.metastore.warehouse.dir":
+              if len(prop.find("value").text.split("/")) == 5: 
```

Comment:
I believe PR #14 and PR #15 should resolve this issue so we can remove this from the current PR